src/main/java/com/suse/matcher/OutputWriter.java

```diff
@@ -45,6 +46,7 @@
     // filenames
     private static final String JSON_INPUT_FILE = "input.json";
     private static final String JSON_OUTPUT_FILE = "output.json";
+    private static final String PROCESSED_JSON_INPUT_FILE = "processed_input.json";
```

Comment:
Please excuse me for being a bit picky, but given we already have an "input" and an "output", I find it philosophically unpleasant to also have a "processed input" - which is by definition an "output"! :wink: :angel: 

Could we make this data part of output.json where it "intuitively belongs"?

Reply:
Make sense. done in the latest commits.